### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Conaclos**
Hi!

The PR brings the next stuffs:
- Improve i18n README
- Add .pot file
- Add .po file for fr_FR

### CODE REVIEW

File: sources/i18n/fr_FR/localization.po [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+msgid "IPv4"
+msgstr "IPv4"
+
+#: views/_ssid.html.php:15
+msgid "Name (SSID)"
+msgstr "Nom (SSID)"
+
+#: views/_ssid.html.php:22
+msgid "Secure"
+msgstr "Sûr"
```
Comment (sbadia) [inline sources/i18n/fr_FR/localization.po]: Hum, we can maybe traduce with something like « sécurisé » no? (the checkbox goal is to enable or not the open hotspot).

File: sources/i18n/fr_FR/localization.po [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+msgid "Hotspot Enabled"
+msgstr "Point d'accès activé"
+
+#: views/settings.html.php:73
+msgid "Device"
+msgstr "Appareil"
+
+#: views/settings.html.php:85
+msgid "Channel"
+msgstr "Cannal"
```
Comment (sbadia) [inline sources/i18n/fr_FR/localization.po]: Typo

File: sources/i18n/fr_FR/localization.po [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+"This is a fast status. Click on More details to show the complete status."
+msgstr "Ceci est un résumé du statut. Cliquez sur Plus de détails pour consulter l'intégralité du statut."
+
+#: views/settings.html.php:39
+msgid "Running"
+msgstr "En cours d'exécution"
+
+#: views/settings.html.php:41
+msgid "Not Running"
+msgstr "Etient"
```
Comment (sbadia) [inline sources/i18n/fr_FR/localization.po]: Typo

**Comment by sbadia** [comment]
Wow! Many thanks @Conaclos !
I just made some inline comments (for typos), otherwise it's ok for me.
Thanks!

**Comment by Conaclos** [comment]
Thanks @sbadia for your feedback.
I've just fixed the typo errors ;)

**Comment by sbadia** [comment]
Thanks ! Sorry to be annoying, but it's possible to squash the five commits ? (I can do this if you want), Thanks in advance :-)

**Comment by Conaclos** [comment]
All should be fine :)

**Pull request merged**